ZLS doesn't compile for latest master

```zig
zls on master via Zig v0.9.0-dev.807+d785dc49a took 21s
❯ zig build -Drelease-safe
./src/known-folders/known-folders.zig:149:38: error: expected type 'type', found '*[8192]u8'
        var line_it = std.mem.split(&read, "\n");
                                     ^
```

--- a/build.zig
+++ b/build.zig
@@ -9,7 +9,10 @@ pub fn build(b: *std.build.Builder) !void {
 
     const mode = b.standardReleaseOptions();
     const exe = b.addExecutable("zls", "src/main.zig");
-    exe.addBuildOption(
+    const exe_options = b.addOptions();
+    exe.addOptions("build_options", exe_options);
+
+    exe_options.addOption(
         []const u8,
         "data_version",
         b.option([]const u8, "data_version", "The data version - 0.7.0, 0.7.1 or master.") orelse "master",

